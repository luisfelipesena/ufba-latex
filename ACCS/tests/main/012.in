5029 34073727 52301466
