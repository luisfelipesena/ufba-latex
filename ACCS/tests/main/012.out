1782105874188811
